activities.json

```diff
@@ -419,6 +419,18 @@
     "title": "Native File System",
     "url": "https://wicg.github.io/native-file-system"
   },
+  {
+    "ciuName": "netinfo",
+    "description": "The Network Information API enables web applications to access information about the network connection in use by the device.",
+    "id": "netinfo",
+    "mozBugUrl": null,
```

Comment:
Maybe https://bugzilla.mozilla.org/show_bug.cgi?id=1057169 ?

Reply:
we could "won't fix" that also.

Reply:
Thanks for the link.  We probably should, but that's a separate decision.

Reply:
I couldn't resist.